src/systems/filecoin_vm/message/message.id

```diff
@@ -16,6 +16,9 @@ type GasAmount struct {
 }
 
 type UnsignedMessage struct {
+    // Version of this message (0 until we have to have a breaking change)
+    Version int64
```

Comment:
Just checking if _int64_ is intentional vis-à-vis _uint64_. Granted, I don't anticipate Filecoin hitting `INT64_MAX`. :)